cmd/release-notes/README.md

```diff
@@ -75,7 +75,7 @@ level=debug timestamp=2019-07-30T04:02:44.3716249Z caller=notes.go:497 msg="Excl
 | github-token | GITHUB_TOKEN | | Yes | A personal GitHub access token |
 | github-org | GITHUB_ORG | kubernetes | Yes | Name of GitHub organization |
 | github-repo | GITHUB_REPO | kubernetes | Yes | Name of GitHub repository |
-| requiredAuthor | REQUIRED_AUTHOR | k8s-ci-robot | Yes | Only commits from this GitHub user are considered. Set to empty string to include all users |
+| required-author | REQUIRED_AUTHOR | k8s-ci-robot | Yes | Only commits from this GitHub user are considered. Set to empty string to include all users |
```

Comment:
This seems a breaking change but I think it might be okay (last famous words).

Reply:
I found, that the flags have a deprecation method and use that now. See https://github.com/kubernetes/release/pull/977/files#diff-91de0979ac440515ffbed7847c8691aeR190-R194.